assets/js/blocks/cart-checkout-shared/payment-methods/payment-method-error-boundary.tsx

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
This will be easier to review (and better for the history) we renamed using `git mv`. Would it be possible to revert this and use git mv instead?